### DISCUSSION TITLE
compatibility with legacy devtools plugin api

### DISCUSSION THREAD

**Discussion by webfansplz** [vuejs/devtools#150] [General]
Due to some third-party libraries using the [legacy devtools plugin api](https://devtools.vuejs.org/plugin/api-reference.html), and the devtools-next doesn't fully compatible with the [legacy devtools plugin api](https://devtools.vuejs.org/plugin/api-reference.html). So we need some feedback to decide if we need to be fully compatible with it.

Please feel free to leave a comment, or approve the plugin you feel needs support in the comments, all feedback is greatly appreciated ♥️.

**Comment by webfansplz** [comment]
[vue-i18n](https://github.com/intlify/vue-i18n-next/blob/master/packages/vue-i18n-core/src/devtools.ts)

**Comment by webfansplz** [comment]
[vue-query](https://github.com/TanStack/query/blob/main/packages/vue-query/src/devtools/devtools.ts)

**Reply by JoostKersjes** [reply]
Here are some related thoughts from @DamianOsipiuk https://github.com/TanStack/query/issues/6632#issuecomment-1914961464

**Comment by webfansplz** [comment]
Released legacy devtools plugin API in v7.3.0
